src/react/deckgl.js

```diff
@@ -118,141 +129,19 @@ export default class DeckGL extends React.Component {
     this.props.onWebGLInitialized(gl);
 
     // Note: avoid React setState due GL animation loop / setState timing issue
-    this.layerManager = new LayerManager({gl});
+    this.layerManager = new LayerManager({gl, canvas});
```

Comment:
I was thinking that the DOM element handling events should not be coupled with the gl context. If event handling exists in its own module, alongside LayerManager (instead of within it), this will be moot anyway (it will accept an arbitrary DOM element just like EventManager does).